.storybook/preview-head.html

```diff
@@ -0,0 +1,3 @@
+<link type="image/x-icon" rel="shortcut icon" href="https://cdn.jsdelivr.net/npm/bulib-wc@latest/dist/icons/favicon.ico">
+<link rel="stylesheet" href="../assets/css/shared.css">
+<link rel="stylesheet" href="./bundle.css">
```

Comment:
note: relying on a copy/paste from within `scripts/build_css.sh` is not the right way to do this longterm. I've created [an issue](https://github.com/open-wc/open-wc/issues/1154) with open-wc to figure out the right one. this is 🆗 in the meantime.